examples/systemcalls/systemcalls.c

```diff
@@ -17,7 +17,13 @@ bool do_system(const char *cmd)
  *   or false() if it returned a failure
 */
 
-    return true;
+  /**
+   * system()
+   *   The value returned is -1 on error, and the return status of the command 
+   *   otherwise
+   */
+  int returnValue = system(cmd);
+  return returnValue == -1 ? false : true;
```

Comment:
@qrogers6 what happens if an error return code other than `-1` is returned by `cmd`?